.github/workflows/_calculate-docker-image.yml

```diff
@@ -25,7 +25,8 @@ jobs:
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
         with:
           submodules: false
-          fetch-depth: 1
+          # A very high bound which should be more than we ever need.
+          fetch-depth: 200
```

Comment:
Hmm, I'm trying to understand why this is needed here. This step fails if the Docker image has been built yet.  But once the image is ready, rerunning it works (that's what works for me)